src/index/etc/di.xml

```diff
@@ -43,4 +43,21 @@
             <argument name="additionalFieldsProvider" xsi:type="object">additionalFieldsProviderForFredhopper</argument>
         </arguments>
     </type>
+
+    <type name="Aligent\FredhopperCommon\Model\Config\CustomAttributeConfig">
```

Comment:
For my reference, does adding _imageurl and _thumburl exclude these attributes from FH export? The reason for asking this question is that there is no indication of exclusion. I may be wrong here.

Reply:
@dominicaligent no - this is to include them in the export. It was missed previously, so they are currently not included.

Reply:
Ok.  Thanks.  I'm just confused with the main description.
`Without a reference in the custom attribute data, _thumburl and _imageurl attributes do not get included in the export to Fredhopper`